functional-tests/test-bed/pom.xml

```diff
@@ -70,7 +70,7 @@
           <forkCount>${surefire.fork.count}</forkCount>
           <threadCount>${surefire.thread.count}</threadCount>
           <reuseForks>true</reuseForks>
-          <argLine>-Xmx1024m -XX:MaxPermSize=256m</argLine>
+          <argLine>-Xmx4096m -XX:MaxPermSize=256m</argLine>
```

Comment:
btw, I think that we don't need `-XX:MaxPermSize=256m`
https://travis-ci.org/arquillian/smart-testing/builds/253109080#L7079